### DISCUSSION TITLE
Column filtering does not work

### DISCUSSION THREAD

**Discussion by jonathanmotif** [AntlerVC/firetable#878] [Support / Q&A] [answered]
When applying a filter should the Value column contain a list of the available values or should it be blank?

I would assume it would contain the values but it does not. See pic. 
<img width="459" alt="image" src="https://user-images.githubusercontent.com/110616581/196435724-150b555f-9a4d-4852-be6e-f755388e1d90.png">

Furthermore, if one types in a valley eg. "Surrey" the "Apply" button is greyed out. 
<img width="510" alt="image" src="https://user-images.githubusercontent.com/110616581/196436233-c87f08c8-7cea-424c-88d0-706a196fb652.png">

Why is that? This is true on Your Filter and Table Filter modes therefore do not function for me.

**Comment by aayaffe** [comment]
I encountered the same issue.
Is this a bug?

**Comment by BeeBombshell** [comment, marked as answer]
Hi @jonathanmotif, @aayaffe. While creating a Single Select type column, make sure to enlist the values to be provided as options. Thereafter, you can select these options from the filter without having to type in the values yourself. If you haven't provided options while configuring the column, that explains the apply button turning grey.

<img width="960" alt="image" src="https://user-images.githubusercontent.com/53828745/198272574-4b84b003-b5ad-4f7a-b815-833c89e93533.png">
